src/gateway/routers/axelar/Router.sol

```diff
@@ -85,6 +87,8 @@ contract AxelarRouter is Auth {
 
     // --- Outgoing ---
     function send(bytes calldata message) public onlyGateway {
-        axelarGateway.callContract(CENTRIFUGE_CHAIN_ID, LP_PRECOMPILE_ADDRESS, message);
+        axelarGateway.callContract(
+            CENTRIFUGE_CHAIN_ID, BytesLib.toHex(abi.encodePacked(centrifugeAxelarExecutable)), message
```

Comment:
Worth noting that this is making every single call more expensive, by adding the abi encoding and hex conversion. I'm not 100% sure that's worth it vs just continuing to keep it hardcoded?

Reply:
Did not know that. Will hardcode